client/platform/desktop/backend/native/viame.ts

```diff
@@ -125,6 +125,13 @@ async function runPipeline(
       command.push(`-s detector_writer:file_name="${detectorOutput}"`);
       command.push(`-s track_writer:file_name="${trackOutput}"`);
     }
+
+    // TODO: Investigate if any other types can use this.
+    if (runPipelineArgs.pipeline.type === 'filter') {
+      command.push(`-s kwa_writer:output_directory="${npath.join(jobWorkDir, 'output')}"`);
+      command.push('-s image_writer:file_name_template="frame%06d.jpg"');
+      command.push('-s image_writer:type="vxl"');
+    }
```

Comment:
This is to accommodate some pipeline args that I noticed were missing. This afaik is not a complete list, but does allow for some pipelines like `filter_to_kwa` to be run.